notifier.py

```diff
@@ -214,7 +214,7 @@ def handle_payload(self, raw):
             if SEND_EMAIL:
                 recipient = ml
                 asfpy.messaging.mail(
-                    sender="GitBox <git@apache.org>",
+                    sender=f"{user} <git@apache.org>",
```

Comment:
It would be nice to have a `displayable user name` instead of `login` here. I guess it is present in the payload.